src/modules/Resize/Resize.js

```diff
@@ -2,7 +2,7 @@ const imagejs = require('imagejs'),
   pixelSetter = require('../../util/pixelSetter'),
   ndarray = require('ndarray');
 module.exports = function Resize(pixels, options) {
-  const resize_value = parseInt(options.resize.slice(0, -1));
+  const resize_value = parseFloat(options.resize);
```

Comment:
So this will take both with and without % right?

Reply:
Yes

Reply:
Ok.. Ty.